app/views/layouts/devise.html.erb

```diff
@@ -0,0 +1 @@
+<%= stylesheet_link_tag "devise2.css" %>
```

Comment:
Getting issues concerning the above line:
![Screen Shot 2019-08-06 at 12 41 45 AM](https://user-images.githubusercontent.com/31193823/62511731-06bcb380-b7e3-11e9-84ad-cefb1aec7825.png)
![Screen Shot 2019-08-06 at 12 41 36 AM](https://user-images.githubusercontent.com/31193823/62511736-0b816780-b7e3-11e9-8c5c-f9ce78f6aaa5.png)

Reply:
Yep, this was a small bug! We have removed this code.